src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/DefaultLSPProjectionProvider.cs

```diff
@@ -17,17 +19,43 @@ internal class DefaultLSPProjectionProvider : LSPProjectionProvider
     {
         private readonly LSPRequestInvoker _requestInvoker;
         private readonly LSPDocumentSynchronizer _documentSynchronizer;
-        private readonly RazorLogger _logger;
+        private readonly ILogger _logger;
+        private Func<RazorLanguageQueryParams, CancellationToken, Task<RazorLanguageQueryResponse>> _inProcLanguageQueryMethod;
 
         [ImportingConstructor]
         public DefaultLSPProjectionProvider(
             LSPRequestInvoker requestInvoker,
             LSPDocumentSynchronizer documentSynchronizer,
-            RazorLogger logger)
+            HTMLCSharpLanguageServerFeedbackFileLoggerProvider logger)
```

Comment:
Why was this changed / is it required? This'll conflict with the changes in my loghub PR.

Reply:
Ah, this was just extras that were left behind after I logged everything to death. I can revert if it'd be problematic

Reply:
If it isn't required, that'd be great. Hoping loghub gets unblocked soon so that we can have all these logs flowing through nicely.